README.md

```diff
@@ -1 +1,19 @@
 # Cin Inside
+
+## Dependencies
+
+* `Java`
+* `Gradle`
+* `Docker` (preferably install via the install script)
+* `Docker Compose`
+* `e-sender`
+* `rabbitmq`
```

Comment:
```suggestion
* `e-sender env file`
```

We can remove RabbitMQ from here, it comes together in the `docker-compose` file, there's no need to install it separately. 
In case of not having the env file, if we disable the e-sender the rest of the system should still run using the commands you attached to the `Run locally` session 😊

Reply:
Great! I'll change it